kafka/tnt_kafka.c

```diff
@@ -25,8 +25,8 @@ luaopen_kafka_tntkafka(lua_State *L) {
             {"poll_rebalances", lua_consumer_poll_rebalances},
             {"store_offset", lua_consumer_store_offset},
             {"close", lua_consumer_close},
+            {"destroy", lua_consumer_destroy},
```

Comment:
We shouldn't do that silently.
Currently it's OK, but please file an issue ("implement __gc metamethod for producers and consumers") and add it to list of "known issues".

Also please reflect this changes in readme example.